LongWarOfTheChosen/Localization/LW_Overhaul.ita

```diff
@@ -1,196 +1,202 @@
+;LWotC Translated
```

Comment:
Sorry, you only need to do this where the `.int` file has a `; LWOTC Needs Translation` block. If you have made other changes to fix existing translations, don't bother marking those.